### PR TITLE
Seungjae/project setup

### LINKED ISSUE #27: Initial Project setup
# Description
-> Project will be set up refer to below project tree. 

- src
├── components                  		each customized component
├───── atoms 
├─────── textField           		grouping Component & Css File 
├────────── TextField.tsx
├────────── TextField.module.css
├───── molecules 
├───── organisms 
├───── templates 
├── constants                           	define constants
├── hooks                            		define custom hooks 
├── services                             	api
├── css                      		global.css, common.css, etc                
├── stores                      		useContext if it’s needed
├── types                      		define types          
├── utils                      		define utils 
├── .env.*                            		build environment
└── package.json               		setting file for project name, script, version of libraries

### PR DESCRIPTION
**Pull request by DLSKAGHD**
Default Project setting 
- Project foleder set 
- Applied reset style
- Add pages folder 
- CommonRouter added for managing routing in one component 
- Mantine UI test by using TextInput

### CODE REVIEW

File: src/components/atoms/TextField.module.css [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.input {
```
Comment (HanKyungSung) [inline src/components/atoms/TextField.module.css]: can it be more specific to reduce the conflict of the variable name here?

File: package.json [hunk 2]
```diff
@@ -12,15 +15,18 @@
     "@types/react-dom": "^18.2.14",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-router-dom": "^6.21.0",
     "react-scripts": "5.0.1",
+    "styled-reset": "^4.5.1",
```
Comment (HanKyungSung) [inline package.json:20]: Would you explain what is this package about?

File: src/constants/enum.ts
Comment (HanKyungSung) [inline src/constants/enum.ts:1]: Would you follow the enum convention from here?

https://www.typescriptlang.org/docs/handbook/enums.html

File: src/global.d.ts
Comment (HanKyungSung) [inline src/global.d.ts:1]: Would you explain what this is about? is this for global styles?

File: package.json (hunk 2 above)
Comment (DLSKAGHD) [reply package.json:20]: styled-reset is for reset default browser style like normalize.css

File: src/global.d.ts
Comment (DLSKAGHD) [reply src/global.d.ts:1]: This is for define type of module.css. If i don't declare type, it throws error.

File: package.json (hunk 2 above)
Comment (HanKyungSung) [reply package.json:20]: I did research about what it does. Please provide more context to it next time.

File: src/components/atoms/TextField.tsx
Comment (HanKyungSung) [inline src/components/atoms/TextField.tsx:1]: I am not too sure if we need the wrapper for the TextField but let's start with this and change it later.

**Review by HanKyungSung — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (DLSKAGHD) [reply package.json:20]: Okay i got it!

**Pull request closed without merging**